bundles/org.palladiosimulator.dataflow.confidentiality.analysis/models/BranchingOnlineShop/My.resourceenvironment

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I would suggest to store the test models in a separate bundle and just reference that one